hubblestack/extmods/returners/splunk_nebula_return.py

```diff
@@ -142,6 +147,8 @@ def returner(ret):
                                 custom_field_value = __salt__['config.get'](custom_field, '')
                                 if isinstance(custom_field_value, str):
                                     event.update({custom_field_name: custom_field_value})
+                                elif isinstance(custom_field_value, unicode):
+                                    event.update({custom_field_name: custom_field_value})
```

Comment:
Instead of adding a new if statement, can we just change the existing one like this?

```
if isinstance(custom_field_value, (str, unicode)):
```